src/auslib/web/public/helpers.py

```diff
@@ -21,4 +24,19 @@ def get_aus_metadata_headers(eval_metadata):
     return headers
 
 
+def get_content_signature_headers(content):
+    headers = {}
+    if app.config.get("AUTOGRAPH_URL"):
+        hash_ = make_hash(content)
+
+        def sign():
+            return sign_hash(
```

Comment:
Now seems like a good time to add a log message here, when we're actually calling out. It should help us gain some insight if we end up with a load spike related to this.

Reply:
I thought it would be nice to see the result also, so logged a little later -- good enough?

Reply:
Yeah, that sounds good too.